README.md

```diff
@@ -52,23 +59,23 @@ The wasms should be located under `target/release/wbuild/litentry-parachain-runt
 
 Similarly, use `make build-runtime-litmus` and `make build-runtime-rococo` to build the litmus-parachain-runtime and rococo-parachain-runtime, respectively.
 
-## launch a standalone node
+## Launch a standalone node
 
 Simply run
 
 ```
 make launch-standalone
 ```
 
-A standalone node will be launched without relaychain, where blocks are finalised immediately. The node is accessible via ws 9944 port.
+A standalone node will be launched without relaychain, where blocks are finalized immediately. The node is accessible via ws 9944 port.
 
-## launch a local network with relaychain + parachain
+## Launch a local network with relaychain + parachain
 
-The following steps take rococo-parachain for example, because `sudo` will be removed for litentry-parachain and [was removed](https://github.com/litentry/litentry-parachain/issues/775) for litmus-parachain. But generally speaking, lauching a local network works with either of the three chain-settings.
+We take rococo-parachain for example, but generally speaking, launching a local network works with either of the three chain-settings.
```

Comment:
I would suggest try to avoid any personal pronoun. (I / we / us / etc. )

Reply:
Noted.